src/views/blocks/parts/details.php

```diff
@@ -73,55 +74,65 @@
+
 			</dd>
 
-			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ) ?>"> <?php esc_html_e( $end_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ); ?>"> <?php esc_html_e( $end_date ); ?> </abbr>
 			</dd>
 
 		<?php
 		// All day (single day) events
 		elseif ( tribe_event_is_all_day() ):
 			?>
 
-			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-datetime published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_date ); ?> </abbr>
 			</dd>
 
 		<?php
 		// Multiday events
 		elseif ( tribe_event_is_multiday() ) :
 			?>
 
-			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Start:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr updated published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_datetime ) ?> </abbr>
+				<abbr class="tribe-events-abbr updated published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_datetime ); ?> </abbr>
+				<?php if ( $show_time_zone ) : ?>
+					<span class="tribe-events-abbr tribe-events-time-zone published"><?php echo esc_html( $time_zone_label ); ?></span>
+				<?php endif; ?>
 			</dd>
 
-			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'End:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ) ?>"> <?php esc_html_e( $end_datetime ) ?> </abbr>
+				<abbr class="tribe-events-abbr dtend" title="<?php esc_attr_e( $end_ts ); ?>"> <?php esc_html_e( $end_datetime ); ?> </abbr>
+				<?php if ( $show_time_zone ) : ?>
+					<span class="tribe-events-abbr tribe-events-time-zone published"><?php echo esc_html( $time_zone_label ); ?></span>
+				<?php endif; ?>
 			</dd>
 
 		<?php
 		// Single day events
 		else :
 			?>
 
-			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ) ?> </dt>
+			<dt> <?php esc_html_e( 'Date:', 'the-events-calendar' ); ?> </dt>
 			<dd>
-				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ) ?>"> <?php esc_html_e( $start_date ) ?> </abbr>
+				<abbr class="tribe-events-abbr tribe-events-start-date published dtstart" title="<?php esc_attr_e( $start_ts ); ?>"> <?php esc_html_e( $start_date ); ?> </abbr>
```

Comment:
* :no_entry_sign: The $text arg must be a single string literal, not "$start_ts".
* :no_entry_sign: The $text arg must be a single string literal, not "$start_date".

<sub>hash: 740d54064e96c0988ddd889b15bb9a82</sub>